src/build.py

```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
```

Comment:
I would use 'prefix=' here and above (or at least be consistent).  'prefix' is the autoconf name for this kind of thing.